src/main/charts/confluence/values.yaml

```diff
@@ -206,6 +206,13 @@ ingress:
 fluentd:
   # -- True if the fluentd sidecar should be added to each pod
   enabled: false
+  # -- True if the helm default config for the fluentd.conf should be used with the values specified below
+  defaultConfig: false
+  # -- True if a custom config should be used for fluentd
+  customConfigFile: false
+  # -- Custom fluent.conf file
+  # fluent.conf: |
+  fluentdConfig: {}
```

Comment:
Is the map necessary here? Is there another key that can be used other than `fluent.conf`?